apps/utils/db-query-checker/env.sample

```diff
@@ -0,0 +1,31 @@
+# Set the tracing/logging level.
+#
+# When running a release build, this will be overridden
+#   by the `tracing` dependency's "release_max_level_" value in `Cargo.toml`.
+#
+# This parameter is optional.
+#RUST_LOG=db_query_checker=trace
+
+# The path to save CSV results to.
+# Note that this will overwrite any file already at that path.
+#
+# This parameter is optional.
+#DB_QUERIES_OUTPUT=results/db_query_checker.csv
+
+# The PostgreSQL DB connection URI,
+#   as documented in <https://www.postgresql.org/docs/11/libpq-connect.html#LIBPQ-CONNSTRING>.
+#
+# Note: The password may need to be percent-encoded, if it contains special characters, e.g.:
+#
+#     $ python -c "import urllib, sys; print urllib.quote(sys.argv[1])" very%cool@password
+#
+# This paramter is required.
+DB_QUERIES_URI=postgres://localuser:insecurelocalpw@localhost:5432/bfd
```

Comment:
I assume that rust supports env vars? having pswd in code can be dicey!

Reply:
It's not technically in code -- this file just sets environment variables. And this particular `env.sample` is fake; it's just meant as a sample. The directory has a `.gitignore` to prevent folks from accidentally committing their real `.env` files.